test/RunTests

```diff
@@ -1503,7 +1503,7 @@ printf '3 |f a b c |e x y z\n2 |f a y c |e x\n' | \
 
 # Test 143: Test 1 using JSON
 {VW} -k -l 20 --initial_t 128000 --power_t 1 -d train-sets/0001.json --json \
-    -f models/0001.model -c --passes 8 --invariant \
+    -c --passes 8 --invariant \
```

Comment:
Why modify the test here?

Reply:
Hi John,

This is because I saw the Appveyor test fail when the same file is written to by different tests.
(perhaps because it supports running tests in parallel)
The file model name `models/0001.model` appears in 3 different tests.

Since to produce the output it is not critical to write a model to disk, and since the model is not used later, I simply removed the `-f ...`. Another possible fix is to give the model a unique name for each separate test.